users/views/user_invitation_view.py

```diff
@@ -0,0 +1,31 @@
+from django.shortcuts import render
+from django.contrib import messages
+from django.contrib.auth.models import User
+from users.forms import UserInvitationForm
+from django.views.generic import View
+from users.models import SignupInvitation
+from django.core.mail import send_mail
+
+
+class UserInvitationView(View):
+    template_name = "registration/send_invitation.html"
+
+    def get(self, request):
+        form = UserInvitationForm()
+        return render(request, self.template_name, {'form': form})
+
+    def post(self, request):
+        form = UserInvitationForm(request.POST)
+        if form.is_valid():
+            invited = SignupInvitation()
+            invited.email_invited = form.data['email_invited']
+            invited.invited_by = User.objects.get(pk=request.user.id)
+            invited.save()
+            link = '/signup/{1}'.format(invited.hash)
```

Comment:
Change the {1} to {}, I got an error on this line...

Reply:
Done